lib/iex/lib/iex/autocomplete.ex

```diff
@@ -269,6 +289,14 @@ defmodule IEx.Autocomplete do
     end
   end
 
+  defp match_map_fields(map, hint) do
+    map
```

Comment:
@myronmarston a for-comprehension is only traversing as well and it will be faster because it will skip all the machinery behind streams that make them lazy. So it is perfect for this use case (when you don't actually need laziness but want to avoid multiple traversals).

Btw, please continue asking questions, regardless of the reason behind them. :)